src/current/cockroachcloud/create-your-cluster.md

```diff
@@ -168,9 +152,24 @@ Otherwise, to enable VPC peering:
       {{site.data.alerts.callout_info}}
       Make sure that you [delete your trial cluster]({% link cockroachcloud/cluster-management.md %}#delete-cluster) before the trial expires. Your credit card will be charged after the trial ends. You can check the validity of the code on the [Billing]({% link cockroachcloud/billing-management.md %}) page.
       {{site.data.alerts.end}}
-1. Click **Create cluster**.
 
-Your cluster will be created in approximately 20-30 minutes.
+## Step 7. Name the cluster
+
+The cluster is automatically given a randomly-generated name. If desired, change the cluster's name. The cluster name must be 6-20 characters in length, and can include lowercase letters, numbers, and dashes (but no leading or trailing dashes). A cluster's name cannot be edited after it is created.
+
+## Step 8. Select the CockroachDB version
+
+When you create a new cluster, it uses the [latest CockroachDB {{ site.data.products.cloud }} production release](https://www.cockroachlabs.com/docs/releases/cloud) by default. All clusters are then upgraded automatically to each subsequent patch release of their major version as it becomes available. To learn more, refer to [Upgrade Policy]({% link cockroachcloud/upgrade-policy.md %}).
```

Comment:
Even for Dedicated, when you create a new cluster using the UI, I think it uses the latest production release, and you can't choose. (unless a Pre-Production Preview is available)